dv/uvm/data/common_sim_cfg.hjson

```diff
@@ -43,8 +44,13 @@
   h: UVM_HIGH
   d: UVM_DEBUG
 
+  // Path to the dut instance (this is used in a couple of places such as coverage cfg
+  // file, xprop cfg file etc. If this is different for your block, then override it with
+  // the 'overrides' directive.
+  dut_instance: "{tb}.dut"
+
   // Default waves dump settings
-  dump_base:  waves
+  dump_file:  waves.{dump}
```

Comment:
I'd rather keep the dump_base/dump_file stuff as was (see notes at vcs_fsdb.tcl).